vulnerabilities/api_v2.py

```diff
@@ -195,7 +195,21 @@ class Meta:
         ]
 
     def get_affected_by_vulnerabilities(self, obj):
-        return [vuln.vulnerability_id for vuln in obj.affected_by_vulnerabilities.all()]
+        """
+        Return a dictionary with vulnerabilities as keys and their details, including fixed_by_packages.
+        """
+        vulnerabilities = obj.affected_by_vulnerabilities.prefetch_related("fixed_by_packages")
```

Comment:
Shouldn't we prefetch `fixed_by_packages` in the viewset itself? I don't think it's a good idea to trigger a fresh query in the serializer, as this would undo query optimization already done in the viewset.